db/migrate/20181123111150_add_consent_to_legal_aid_applications.rb

```diff
@@ -0,0 +1,6 @@
+class AddConsentToLegalAidApplications < ActiveRecord::Migration[5.2]
+  def change
+    add_column :legal_aid_applications, :open_banking_consent, :string
+    add_column :legal_aid_applications, :consent_choice_timestamp, :datetime
```

Comment:
The rails convention is to use the `_at` suffix with datetime fields. Also the consent type being timestamped should be identified. So perhaps: `open_banking_consent_given_at`

Reply:
I initially used open_banking_consented_at but that implied consent and as consent can be denied I revised the name. I will rethink it.